### PR TITLE
ZCS-11691: Cannot send meeting with message attachment(.msg) using mo…

### PR DESCRIPTION
**Pull request by gabhinavTI**
Issue: Cannot send meeting with message attachment(.msg) using modern UI(tika-parsers issue)

Whenever we attach a .msg file while sending an invite it would fail with an error
`ua=ZimbraXWebClient - GC103 (Windows)/9.1.0_BETA_4334;soapId=7e9080de;] SoapEngine - handler exception java.lang.NoClassDefFoundError: org/apache/james/mime4j/codec/DecodeMonitor at org.apache.tika.parser.microsoft.OutlookExtractor.decodeHeader(OutlookExtractor.java:563) ~[tika-parsers-1.24.1.jar:1.24.1] at org.apache.tika.parser.microsoft.OutlookExtractor.normalizeHeaders(OutlookExtractor.java:529) ~[tika-parsers-1.24.1.jar:1.24.1] at org.apache.tika.parser.microsoft.OutlookExtractor.parse(OutlookExtractor.java:161) ~[tika-parsers-1.24.1.jar:1.24.1] at org.apache.tika.parser.microsoft.OfficeParser.parse(OfficeParser.java:199) ~[tika-parsers-1.24.1.jar:1.24.1] at org.apache.tika.parser.microsoft.OfficeParser.parse(OfficeParser.java:131) ~[tika-parsers-1.24.1.jar:1.24.1] at org.apache.tika.parser.CompositeParser.parse(CompositeParser.java:280) ~[tika-core-1.24.1.jar:1.24.1]`

Basically upon checking `DecodeMonitor` class was missing. This class is available in `org.apache.james`. Adding this resolved the issue. Raised PR in zm-zcs-lib : https://github.com/Zimbra/zm-zcs-lib/pull/101

Testing: Manual testing to send invites by adding .msg attachment and it's successfully sent

### CODE REVIEW

**Comment by silentsakky** [comment]
if I understand correctly then tika parser is used to parse attachments and create indexes for search, why do we need to parse message in message attachments?

**Comment by gabhinavTI** [comment]
> if I understand correctly then tika parser is used to parse attachments and create indexes for search, why do we need to parse message in message attachments?

@silentsakky the attachments here is `.msg `file. Files with .`msg` attachment. Without this dependency, the parsing of this file attachment would fail hence we were getting error. As per current code logic, all the attachments get parsed

**Review by silentsakky — APPROVED** [review]

**Review by desouzas — APPROVED** [review]
